### DISCUSSION TITLE
nb_cli创建bot项目后无法启动

### DISCUSSION THREAD

**Discussion by PaluAix** [nonebot/discussions#440] [问答] [answered]
![image](https://github.com/nonebot/discussions/assets/146350568/5c9917f9-b936-4270-b42b-05a396e55869)
下面是报错
![image](https://github.com/nonebot/discussions/assets/146350568/703f2eb8-32e5-4ffb-8f75-fafd0a4cbc15)

**Comment by RF-Tar-Railt** [comment, marked as answer]
qq适配器需要正向http和ws的驱动器
你应该把fastapi去掉，选择aiohttp驱动器，或者选择httpx+websockets适配器
